clients/admin-ui/src/features/common/table/v2/FidesTable.tsx

```diff
@@ -73,10 +77,15 @@ const HeaderContent = <T,>({
         as={Button}
         rightIcon={<ChevronDownIcon />}
         variant="ghost"
+        size="sm"
+        height={9} // same as table header height
         width="100%"
-        pr={1}
+        sx={{ ...getTableTHandTDStyles(header.column.id) }}
         textAlign="start"
         data-testid={`${header.id}-header-menu`}
+        _focus={{
+          backgroundColor: "gray.100",
+        }}
```

Comment:
```suggestion
        _focus={{
          outline: "none",
        }}
```

Reply:
with this change, a keyboard user has no indication where the focus is, which is why I added bg color, which matches the hover state. Could still hide the outline.

```suggestion
        _focus={{
          backgroundColor: "gray.100",
          outline: "none",
        }}
```